app/main/views/services.py

```diff
@@ -199,7 +198,7 @@ def service_updates(service_id):
         all_update_events=all_update_events,
         latest_update_events=latest_update_events,
         oldest_update_events=oldest_update_events,
-        n_editing_users_min=len(frozenset(audit_event["user"] for audit_event in chain(
+        number_of_users_who_made_edits=len(frozenset(audit_event["user"] for audit_event in chain(
```

Comment:
This isn't the number of editing users - this is the *minimum* possible number of editing users, as we're only looking at the first & last pages of edits, hence the original name.

Reply:
Good call, I changed the name of this variable and added a comment. Still, as per the comment, there would need to be a whole lot of unapproved edits for this code to stop showing the definite total number of users who made unapproved edits.

Reply:
Could easily happen (who knows when someone will decide to change the default page length for calls like this), and when it does happen it will quite happily show a number that is orders of magnitude wrong.

Or very happily make it look like all edits were done by a single (possibly admin?) user when they weren't.